check_run/www/print_check_run.py

```diff
@@ -6,12 +6,13 @@
 	get_print_style,
 	escape_html,
 )
-
+from frappe.www.printview import set_link_titles, get_rendered_template, get_print_format_doc
 
 from check_run.check_run.doctype.check_run.check_run import get_check_run_settings
 
 
 def get_context(context):
+	frappe.msgprint("hello")

```

Comment:
This didn't show up for me during testing, but I think we should remove this

Reply:
yes